kernel/Dockerfile.bcc

```diff
@@ -107,6 +117,6 @@ FROM scratch
 ENTRYPOINT []
 CMD []
 WORKDIR /
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/lib64
```

Comment:
The dockerfile seems to copy stuff which was built for `lib64` to `lib` in the output, so is this really needed? Would be better to arrange for it not to be in any case.
(for a future change though if you prefer)

Reply:
this was actually in the previous stage, where i made no sense to me, so I just moved it to the final stage. Can probably be removed...Will do when I get around to switching the kernel build to `linuxkit pkg build`